packages/react-core/src/components/LabelGroup/examples/LabelGroup.md

```diff
@@ -104,6 +128,81 @@ VerticalCategoryLabelGroupOverflowRemovable = () => {
         </Label>
       ))}
     </LabelGroup>
-  )
+  );
+};
+```
+
+### Editable labels
+
+```js
+import React from 'react';
+import { LabelGroup, Label, TextArea } from '@patternfly/react-core';
+
+class EditableLabelGroup extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      label1: 'Editable label',
+      label2: 'Editable label 2',
+      label3: 'Editable label 3'
+    };
+    this.onEditCancel = (prevText, label) => {
+      this.setState({
+        [label]: prevText
+      });
+    };
+    this.onEditComplete = (newText, label) => {
+      this.setState({
+        [label]: newText
+      });
+    };
+  }
+
+  render() {
+    return (
+      <LabelGroup numLabels={5} isEditable>
+        <Label
+          color="blue"
+          onClose={Function.prototype}
+          onEditCancel={prevText => this.onEditCancel(prevText, 'label1')}
+          onEditComplete={newText => this.onEditComplete(newText, 'label1')}
+          isEditable
+          editableProps={{
+            'aria-label': 'Editable text',
+            id: 'editable-label-1'
+          }}
+        >
+          {this.state.label1}
+        </Label>
+        <Label color="green">Static label</Label>
```

Comment:
Is there a reason to make this a different color? At first glance someone may think the color indicates whether the label is editable or static.

Reply:
I was matching core's example, should I change this to blue?

Reply:
It didn't bother me that these are different. It's true that the color is not meaningful here, but it might be a valid way to differentiate them.